### ISSUE TITLE
Anyone (other than author) get this working locally or on Google collab?

### ISSUE THREAD

**Issue by BlockchainPunks** [aashishrai3799/Towards-Realistic-Generative-3D-Face-Models#20]
anyone other than the author manage to get this working on Linux or google collab yet? I’m still sceptical about this project as we have had many failures getting it functional?

if you did - can you share your working repo with all relevant files folders scripts etc via link?

**Comment by paarthmadan29** [comment]
Yes, I have been able to run it successfully without any errors. I am also planning to create a readme as well. I will try to upload and make a PR by this weekend. 
Best,

**Comment by misakamage** [comment]
Hi, I successfully ran this demo on Colab; however, my results, both in terms of texture and mesh, are unsatisfactory. The Colab notebook can be found at https://colab.research.google.com/drive/18YUcr1lIfSnxSlZiaND0tvt5FILXMjcm?usp=sharing. If you're interested, we can work on it together.

**Comment by BlockchainPunks** [comment]
> Hi, I successfully ran this demo on Colab; however, my results, both in terms of texture and mesh, are unsatisfactory. The Colab notebook can be found at https://colab.research.google.com/drive/18YUcr1lIfSnxSlZiaND0tvt5FILXMjcm?usp=sharing. If you're interested, we can work on it together.

Excellent - finally someone who knows how to set this up correctly! - I’ll reach out to you in the new year - travelling overseas until 10’th Jan 2024

**Comment by misakamage** [comment]
> Excellent - finally someone who knows how to set this up correctly! - I’ll reach out to you in the new year - travelling overseas until 10’th Jan 2024

OK, hope you have a good time!

**Comment by Garv62003** [comment]
> Hi, I successfully ran this demo on Colab; however, my results, both in terms of texture and mesh, are unsatisfactory. The Colab notebook can be found at https://colab.research.google.com/drive/18YUcr1lIfSnxSlZiaND0tvt5FILXMjcm?usp=sharing. If you're interested, we can work on it together.

Hi ! I am Garv, 
I have been working on this rep for about a week but am not able to set it up can You please help me with this, I tried using the code you provided but it throws me some errors I would really be helpful if we can help me out 
my contact - garvgupta.iitd@gmail.com
